Z/z3/build_tarballs.jl

```diff
@@ -36,9 +35,12 @@ fi
 """
 
 # Bash recipe for building across all platforms
+# Patches Z3 to work around https://github.com/Z3Prover/z3/issues/6890 (closed but not fixed!)
 script = macfix * raw"""
 cd $WORKSPACE/srcdir/z3-*/core
 
+sed -i '16,30d' src/util/memory_manager.cpp
```

Comment:
That's rather opaque and brittle, isn't it? If someone else wants to update the recipe in the future, they'll first have to reverse engineer what exactly this does.

Can't you instead e.g. specify the exact string to be deleted? Or provide an actual patchfile?

Reply:
Agreed, pushed a more explicit command and linked to a new issue documenting the bug.